src/main/resources/application.yml

```diff
@@ -9,16 +9,18 @@ spring:
   application:
     name: jwt-demo
   datasource:
-      username: jwtuser
-      password: letmein
-      url: jdbc:postgresql://localhost:5432/jwt-db
+    username: jwtuser
+    password: aFFsaToBgb1PXTrn2xAZHmBBK9zAcTc0
+    url: jdbc:postgresql://dpg-cpfmqu7109ks73bnivv0-a.oregon-postgres.render.com/jwt_db_o1fj
```

Comment:
@AbdullatifHabiba 
This is real DB connection host, I assumed this by fault, isn't it?